client/stylesheets/shared/_reset.scss

```diff
@@ -64,95 +66,40 @@
 	display: none;
 }
 
-.wp-responsive-open {
-	.woocommerce-layout__header {
-		margin-left: 2px;
+.woocommerce_page_wc-admin {
+	.wp-responsive-open {
+		.woocommerce-layout__header {
+			margin-left: 2px;
+		}
 	}
 
-	@include breakpoint( '<782px' ) {
-		.woocommerce-layout__activity-panel-wrapper.is-open,
-		.woocommerce-layout__activity-panel-tabs {
-			width: calc(100vw - 190px);
-		}
+	.components-popover:not(.is-mobile) {
+		z-index: 1 !important; /* below of woocommerce-layout__header */
```

Comment:
Is the `!important` necessary here? I think this selector will already be more specific that the Gutenberg one.

Reply:
Yes, good catch! Fixed in 825f76e13c154a0afc08a2529c72ca6e736beda2.